taxcalc/current_law_policy.json

```diff
@@ -943,6 +943,32 @@
         "validations": {"min": 0, "max": 1}
     },
 
+    "_CG_dollar_ec": {
```

Comment:
This all makes sense to me,  including separating the other parameter name fixes into a new PR. We will want to issue a release before that PR is merged as it will have breaking changes that require coordinating with tax-calculator users.  (cc @talumbau). 

@martinholmer, do you think our current use of "floor" is appropriate? It seems to be a special case of an exclusion where the excluded amount is keyed off of some special income type. 

(cc @Amy-Xu on this discussion as well).